apps/web-giddh/src/app/app.constant.ts

```diff
@@ -441,3 +441,15 @@ export const JOURNAL_VOUCHER_ALLOWED_DOMAINS = [
     'muneem.co',
     'whozzat.com',
 ];
+
+/**
+ * Enum for switching toggle button On and Off and changing its size
+ *
+ * @export
+ * @enum {string}
+ */
+export enum BOOTSTRAP_TOGGLE_SWITCH {
```

Comment:
Enums are always named in CamelCase
```suggestion
export enum BootstrapToggleSwitch {
```